### ISSUE TITLE
[Bug] FakeNitro doesn't ignore "use external emoji/stickers" permissions in sticker picker

### ISSUE THREAD

**Issue by xparadoxical** [Vendicated/Vencord#2173]
tldr: emoji bypass bypasses permissions, sticker bypass does not.

### Discord Account

230v

### What happens when the bug or crash occurs?

FakeNitro respects "use external emoji" and "use external stickers" permissions, not letting you send any sticker you want. Occurs in any place where you can send messages that has the two permissions restricted in any combination.

| Use external emoji | Use external stickers | Server icons<br/>in the sticker picker | Favorite<br/>stickers | Most used<br/>stickers | Other external<br/>stickers |
|:-:|:-:|:-:|:-:|:-:|:-:|
| ✔️ | ✔️ | visible | enabled | enabled | enabled |
| ✔️ | ❌ | visible | **disabled** | **hidden** | **disabled** |
| ❌ | ✔️ | **hidden** | enabled | enabled | **hidden** |
| ❌ | ❌ | **hidden** | **disabled** | **hidden** | **hidden** |

Bold items should be changed.

Stable 265637 (863f550)
Vencord 8b6a403 (Vesktop V1.5.0)
Electron 28.1.3
Chromium 120.0.6099.199

Stable 265637 (863f550)
Vencord 8b6a403 (Web)

![image](https://github.com/Vendicated/Vencord/assets/43796685/27a407c2-16c2-4f14-ac92-8feba8572206)

### What is the expected behaviour?

Being able to get/send links to stickers from any server no matter what's the state of the two permissions, just like with the "Emoji bypass" option. The restriction doesn't make much sense (thus I perceive it as a bug) as it can be trivially worked around by getting annoyed, going to a different channel for a second, getting the link there and sending it in the original channel.

| Use external emoji | Use external stickers | Server icons<br/>in the sticker picker | Favorite<br/>stickers | Most used<br/>stickers | Other external<br/>stickers |
|:-:|:-:|:-:|:-:|:-:|:-:|
| ✔️ | ✔️ | visible | enabled | enabled | enabled |
| ✔️ | ❌ | visible | **enabled** | **enabled** | **enabled** |
| ❌ | ✔️ | **visible** | enabled | enabled | **enabled** |
| ❌ | ❌ | **visible** | **enabled** | **enabled** | **enabled** |

### How do you recreate this bug or crash?

1. find a channel with either of the permissions disabled (use PermissionsViewer), or create a test server on an alt account and invite a non-owner account that uses vencord
2. disable one or both of the two aforementioned permissions in a text channel, voice channel messages or a thread
3. open the sticker picker

### Errors

none

### Request Agreement

- [X] I am using Discord Stable or tried on Stable and this bug happens there as well
- [X] I have read the requirements for opening an issue above

**Comment by Nuckyz** [comment]
Fixed in another commit where the settings that adds these bypasses were explained better
